components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -1205,7 +1205,7 @@ spec:
       containers:
       - args:
         - -zap-log-level=6
-        image: quay.io/redhat-appstudio/pipeline-service-exporter:2a5d2420d60643d685d860e3aa36824de715b9b8
+        image: quay.io/redhat-appstudio/pipeline-service-exporter:2782659983c94692497467cd1cf952b1bc1f0da4
```

Comment:
this tag exists in https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags